package.json

```diff
@@ -16,8 +16,9 @@
     "babel": "rimraf lib && babel --stage 0 modules/ -d lib/",
     "dist": "webpack && webpack --config=webpack.config.minified.js",
     "examples": "webpack-dev-server --config examples/webpack.config.js --no-info --content-base examples/",
+    "flow": "node -e \"process.exit(process.platform === 'win32' ? 0 : 1)\" || flow check",
     "lib": "npm run babel && rimraf lib/__tests__ lib/__mocks__",
-    "lint": "eslint modules",
+    "lint": "eslint modules && npm run flow",
```

Comment:
We should have a slightly separate path for dev vs CI then; I don't mind flow being part of lint, but `flow check` is far too slow to be part of lint. `flow status`, which starts the server, is much much faster.